docs/ref/contrib/gis/geoip2.txt

```diff
@@ -0,0 +1,199 @@
+=======================
+Geolocation with GeoIP2
+=======================
+
+.. module:: django.contrib.gis.geoip2
+   :synopsis: High-level Python interface for MaxMind's GeoIP C library.
+
+The :class:`GeoIP2` object is a wrapper for the
+`MaxMind geoip2 Python library`__. [#]_
+
+In order to perform IP-based geolocation, the :class:`GeoIP2` object requires
+the libmaxminddb C library, the geoip2 Python library and either the GeoIP
+`Country` or `City` datasets in binary format (the CSV files will not work!).
+These datasets may be `downloaded from MaxMind`__.  Grab the
+``GeoLite2-Country.mmdb.gz`` and `` GeoLite2-City.mmdb.gz`` files and unzip them
+in a directory corresponding to what you set :setting:`GEOIP_PATH` with in your
+settings.  See the example and reference below for more details.
+
+__ https://github.com/maxmind/libmaxminddb
+__ http://geoip2.readthedocs.org/
+__ http://dev.maxmind.com/geoip/geoip2/geolite2/
+
+Example
+=======
+
+Assuming you have the libmaxminddb C library installed, here is an example of its
```

Comment:
apparently it's not. If the C library is installed, the `maxminddb` python library will use that (and supposedly be faster). Otherwise, it falls back on pure Python implementation.